WinRT.Runtime/ApiCompatBaseline.net5.0.txt

```diff
@@ -1 +1,3 @@
-Total Issues: 0

```

Comment:
I was initially a bit confused why this was considered a breaking API compat change as the change is adding a default interface method which I would assume should be fine from a compat perspective.  From looking into it, the rules for default interface methods in the api compat tooling are not enabled by default due to it seems like it might not be fully supported by VB and might affect consumers with LangVersion before 8.

I don't think either of those should affect us and due to that, I think we can enable the optional rules for default interface members by adding the below to our winrt.runtime.csproj and then this shouldn't get flagged anymore.

```xml
    <ApiCompatArgs>$(ApiCompatArgs) --allow-default-interface-methods</ApiCompatArgs>
    <MatchingRefApiCompatArgs>$(MatchingRefApiCompatArgs) --allow-default-interface-methods</MatchingRefApiCompatArgs>
```

Reply:
that works - thx